geometry_msgs/Point[] array